semcore/radio/src/Radio.jsx

```diff
@@ -48,102 +51,216 @@ class RadioGroupRoot extends Component {
 
 const RadioGroup = createComponent(RadioGroupRoot);
 
-class Radio extends Component {
+class RadioRoot extends Component {
   static displayName = 'Radio';
   static style = style;
   static contextType = RadioGroup[CONTEXT_COMPONENT];
 
-  bindHandlerChange = (value) => (checked, e) => {
-    this.context.onChange(value, e);
+  // static defaultProps = {
```

Comment:
Why did you removed default state and size?

Reply:
We should get it from context but could redefine them in component.
So, I set defaults in `get*Props` if there are no one of them in context or props of `Radio`, because otherwise context couldn't define them